.gitignore

```diff
@@ -0,0 +1,2 @@
+# Generated dart files
```

Comment:
Shouldn't the gitignore file contain much more than just that?

Reply:
Yes but by default, Flutter project are made of multiple. So this is ours at root level and we have many more. We concluded that it should stay like that last Pull Request.

See https://github.com/nventive/FlutterApplicationTemplate/pull/1#discussion_r1519837368 for more details.